Packs/Stamus/Integrations/Stamus/Stamus.yml

```diff
@@ -0,0 +1,109 @@
+category: Network Security
+commonfields:
+  id: Stamus
+  version: -1
+configuration:
+- defaultvalue: https://example.com/
+  display: Stamus Central Server
+  name: url
+  required: true
+  type: 0
+- displaypassword: API Key
+  additionalinfo: The API Key to use for connection
+  name: credentials
+  required: true
+  hiddenusername: true
+  type: 9
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- defaultvalue: '200'
+  display: Maximum number of incidents per fetch
+  name: max_fetch
+  required: false
+  type: 0
+  section: Collect
+- defaultvalue: 3 days
+  display: First fetch time
+  name: first_fetch
+  required: false
+  type: 0
+  section: Collect
+description: '[Get Declaration of Compromises from Stamus Security Platform and build Incidents. Then get related artifacts, events and Host Insight information]'
+display: 'Stamus'
+name: Stamus
+script:
+  commands:
+  - arguments:
+    - name: key
+      default: false
+      description: '[Indicator of Compromise key]'
+      isArray: false
+      required: true
+      secret: false
+    - name: value
+      default: false
+      description: '[Indicator of Compromise value]'
+      isArray: false
+      required: true
+      secret: false
+    deprecated: false
+    description: '[Get events with IOC key/value filter]'
+    execution: false
+    name: Stamus-Check-IOC
+    outputs:
+    - contextPath: StamusIntegration.IOC
+      description: '[Fetch events matching an IOC.]'
+      type: String
+  - arguments:
+    - default: false
+      description: '[IP to get Host Insights information]'
+      isArray: false
+      name: ip
+      required: true
+      secret: false
+    deprecated: false
+    description: '[Get Host Insights information]'
+    execution: false
+    name: Stamus-Get-Host-Insight
+    outputs:
+    - contextPath: StamusIntegration.HostInsights
+      description: '[Fetch information about a host known by Host Insight module]'
+      type: String
+  - arguments:
+    - default: false
+      description: '[Stamus ID used to get related information]'
+      isArray: false
+      name: id
+      required: true
+      secret: false
+    deprecated: false
+    description: '[Get events for a Declaration of Compromise using the Stamus ID]'
+    execution: false
+    name: Stamus-Get-DoC-Events
+    outputs:
+    - contextPath: StamusIntegration.RelatedEvents
+      description: '[Get events for a Declaration of Compromise.]'
+      type: String
```

Comment:
please add full context paths for each api response field if possible.

you can take a look at other integrations to see examples.

Reply:
I'm struggling with this.

For example, the stamus-get-host-insight command is giving me a result similar to this when displayed:
![image](https://github.com/demisto/content/assets/308616/93603296-3354-45ee-83dc-b7a06e57e735)

This is an history of a host with all first time we have seen a service, an tls/http agent seen on the IP, an hostname, ...

I got some question:
- Should I put the timestamp in the contextPath ?
- Is the data structure correct for XSOAR ? I can not really set a contextPath to an event because it is a combination of `type` and `value`. Like `type:hostname` will potentially give me an interesting evidence but it is completely different of an information like `type:service` which tell me a service has been seen

Reply:
* yes, you can definitely put timestamp in the contextPath in the yml, the type should be `Date`
*  this data structure looks good to me, you can explain very briefly in the description of each context path what are the possible informations as long as they are on the same contextPath. if your api response would look something like this for example:

```
[
    {
        "timestamp": "some timestamp"
        "ip": "some ip",
        "type": "some type"
        "value": some value
    }
]
```

then your context path would look something like:

```
    outputs:
    - contextPath: Stamus.HostInsight.timestamp
      description: "some description what it means.."
      type: Date
   - contextPath: Stamus.HostInsight.ip
      description: "some description what it means.."
      type: String
   - contextPath: Stamus.HostInsight.type
      description: "some description what it means.."
      type: String
   - contextPath: Stamus.HostInsight.value
      description: "some description what it means.."
      type: String
```